server/src/services/folder_apps.service.ts

```diff
@@ -11,6 +11,14 @@ export class FolderAppsService {
   ) {}
 
   async create(folderId: string, appId: string): Promise<FolderApp> {
+    const oldFolderApp = await this.folderAppsRepository.findOne({
```

Comment:
shall we rename the variable something like `existingFolderApp`? `old` will be confusing.

Reply:
Yes! it should be more informative